2.x/SCO/x-oca-asset.md

```diff
@@ -0,0 +1,289 @@
+## x-oca-asset SCO object
+The `x-oca-asset` object is a component of telemtry that describes an asset and its properties. 
+It can be referenced to give context on events that took place on this asset.
+It describes the hostname, the associated IP addresses and physical addresses of the asset.
+In SIEM terminology which collects events from many systems - it is used to describe where the events took place.
+It is referenced in the [`x-oca-event`](./x-oca-event.md) object.
+
+While `x-oca-asset` specifies the host on which the associated event occurred, it may provide extenstions to describe addtional entities of tne runtime environment. In a cloud environment the event may occur within a container that is running or a partcular pod on the host. `x-oca-asset` uses a separate extension for each addtional runtime entity. Please note that as per the STIX [2.0](http://docs.oasis-open.org/cti/stix/v2.0/cs01/part3-cyber-observable-core/stix-v2.0-cs01-part3-cyber-observable-core.html#_Toc496715388)and [2.1](https://docs.oasis-open.org/cti/stix/v2.1/csprd01/stix-v2.1-csprd01.html#_Toc16070830), COOs/SCOs can have any number of extenstions.
+
+| property name | type | description |
+|--|--|--|
+| **type** (required) | `string` | x-oca-asset |
+| **id** (required) | `string` | A valid [stix-id](https://docs.oasis-open.org/cti/stix/v2.1/os/stix-v2.1-os.html#_64yvzeku5a5c) based on`x-ibm-ttp-tagging` |
+| **spec_version** (optional) | `string` | The value of this property MUST be 2.1 for STIX Objects defined according to this specification. |
+| **extensions** | `dictionary` | Specifies any extensions of the object, as a dictionary. |
+| **device_id** | `string` | The ID of the device. |
+| **hostname** (required)|`string`|name of this host|
```

Comment:
It's minor, but should ensure that the descriptions are either all capitalized or none are.

Reply:
Thanks for pointing that out, there should be a preposition as well, since thats how most descriptions start. So i made it
"The name of the host"